mocca-client/src/test/java/com/paypal/mocca/client/MoccaClientQueryTest.java

```diff
@@ -58,6 +60,34 @@ public void queryRawTest() {
         assertEquals(result.getBar(), "far");
     }
 
+    @Test
+    public void queryInvalidRequest() {
+        try {
+            client.getOneValidSample(new ValidatedRequestDTO(null, "zazzzzzzzzzzzzzzzzzzzzzzzzzz"));
+            fail("should throw a validation exception");
+        } catch (EncodeException e) {
+            System.out.println(((ConstraintViolationException) e.getCause()).toString());
```

Comment:
Is this line intentional? Or did you forget to remove it?

Reply:
I removed it.